.goreleaser.yaml

```diff
@@ -8,8 +8,8 @@ builds:
       - CGO_ENABLED=0
     goos:
       - linux
-#      - windows
-#      - darwin
+      - windows
```

Comment:
any particular reason these were previously commented?

Reply:
not really, just wanted to start with similar build metacontroller had in previous approach